rcgen/Cargo.toml

```diff
@@ -47,8 +47,9 @@ allowed_external_types = [
 
 [dev-dependencies]
 openssl = "0.10"
+pki-types = { package = "rustls-pki-types", version = "1" }
```

Comment:
Is there a reason you wanted to avoid using the `rustls-webpki::types` re-export?

Reply:
I sort of forgot it existed, but now that you brought it up, I think we decided when merging the rustls re-export that the rustls-webpki was maybe not a great fit and that we should remove it in a future release?

Reply:
Ah yes, I remember that. We also talked about how rustls re-exports it as "pki-types" and webpki chose "types".

With that context in mind I think not using the webpki re-export makes sense :+1: